kubeflow/common/asm/Makefile

```diff
@@ -38,4 +38,12 @@ install-asm: downalod-install-asm-script download-asm-package
 	--cluster_location $(LOCATION) \
 	--mode install \
 	--enable_all \
-	--custom_overlay ./asm/istio/options/iap-operator.yaml 
\ No newline at end of file
+	--custom_overlay ./asm/istio/options/iap-operator.yaml 
```

Comment:
From ASM team's response, I think we can do the following in one command (repeat custom_overlay argument multiple times to include multiple overlays):

```
...
--custom_overlay ./asm/istio/options/iap-operator.yaml \
--custom_overlay ./options/ingressgateway-annotations.yaml
```

can you confirm?

Reply:
That is a good point! I wasn't successful in installing multiple overlays previously. But later on I realized that my command syntax was not right. After update, I am able to use multiple overlays in one command.